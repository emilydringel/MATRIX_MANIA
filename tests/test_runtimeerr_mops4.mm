def int main(){
    matrix<float> m1 = [.1,.2,.3;.4,.5,.6];
    matrix<float> m3 = [1.1,2.2;3.3,4.4;5.5,6.6];
    printmf(m1+m3);
}