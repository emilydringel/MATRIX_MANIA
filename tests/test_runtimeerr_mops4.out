RUNTIME ERROR: matrices being added do not have the same dimensions.